src/rgw/rgw_civetweb_frontend.cc

```diff
@@ -77,13 +77,6 @@ int RGWCivetWebFrontend::run()
   }
   conf_map.emplace("listening_ports", std::move(listening_ports));
 
-  /* Set run_as_user. This will cause civetweb to invoke setuid() and setgid()
-   * based on pw_uid and pw_gid obtained from pw_name. */
-  std::string uid_string = g_ceph_context->get_set_uid_string();
-  if (! uid_string.empty()) {
-    conf_map.emplace("run_as_user", std::move(uid_string));
-  }
-
```

Comment:
`mg_start()` is doing several things after its call to `set_uid_option()` - and while it doesn't look like any of it depends on the new uid/gid, that could change upstream in the future. so without better test coverage of its options, i'd prefer not to change how the civetweb side works